harmonic/install_windows_src.md

```diff
@@ -132,6 +142,34 @@ call install\setup.bat
 This is the end of the source install instructions; head back to the [Getting started](/docs/all/getstarted)
 page to start using Gazebo!
 
+> [!WARNING]
+> As Gazebo GUI is not yet working, running `gz sim` will not work. You can run only the server with
+
+```cmd
+gz sim -s -v
+```
+
+> [!WARNING]
+> If you username contains spaces (which is quite common on Windows), you will probably get errors
+  saying `Invalid partition name [Computer:My User With Spaces]`. Fix this by changing `GZ_PARTITION`
+  to something else:
+
+```cmd
+set GZ_PARTITION=test
+```
+
+Remember to set the same partition in all other consoles.
```

Comment:
Same as above.
And all the lines need to be indented.
With the suggested change, result looks like this:
![Screenshot from 2023-09-15 21-20-16](https://github.com/gazebosim/docs/assets/7608908/dd2b41b6-9729-4c44-ac8b-8741dbc63ba3)

```suggestion
> **WARNING**
> If your username contains spaces (which is quite common on Windows), you will probably get errors
>  saying `Invalid partition name [Computer:My User With Spaces]`. Fix this by changing `GZ_PARTITION`
>  to something else:
> ```cmd
> set GZ_PARTITION=test
> ```
> Remember to set the same partition in all other consoles.
```

Reply:
I would suggest to use a word other than `WARNING` though, because that's rather severe, and to differentiate from the yellow warning bar up top.
It sounds more like a tip or a troubleshoot to me. What makes these WARNINGs different from a NOTE? Can they just be NOTEs?